Extensions/notificationblock.js

```diff
@@ -29,303 +29,168 @@ XKit.extensions.notificationblock = new Object({
 
 	},
 
-	run: function() {
+	run: async function() {
 
 		this.running = true;
 
 		if (XKit.interface.where().inbox === true) {
 			return;
 		}
 
+		await XKit.css_map.getCssMap();
 		XKit.tools.init_css("notificationblock");
 
-		try {
-			var m_blacklist = XKit.storage.get("notificationblock", "posts", "").split(",");
-			if (m_blacklist !== "") {
-				this.blacklisted = m_blacklist;
-			}
-		} catch (e) {
-			XKit.storage.set("notificationblock", "posts", "");
-			this.blacklisted = [];
```

Comment:
fun fact: because of the parseInt on https://github.com/new-xkit/XKit/blob/master/xkit.js#L437, i'm can't see how notificationBlock ever worked. Like, at all.